lib/chef/web/core/helpers.rb

```diff
@@ -0,0 +1,34 @@
+require 'chef/web/core'
+require 'chef/web/core/component'
+require 'nokogiri'
+
+class Chef
+  module Web
+    module Core
+      module Helpers
```

Comment:
Ok. I just followed what was in `oc`, but `AssetHelpers` seems good, too. I'll do that.